WordPress/UITestsFoundation/Screens/ActivityLogScreen.swift

```diff
@@ -1,8 +1,9 @@
+import ScreenObject
 import XCTest
 
-public class ActivityLogScreen: BaseScreen {
+public class ActivityLogScreen: ScreenObject {
 
-    public init() {
-        super.init(element: XCUIApplication().otherElements.firstMatch)
+    public init(app: XCUIApplication = XCUIApplication()) throws {
+        try super.init(expectedElementGetters: [ { $0.otherElements.firstMatch } ])
```

Comment:
Worth pointing out that this matches is pretty useless because it will succeed most of the time.

The aim of the current work is not to fix "issues" in the tests, but to move to `ScreenObject` to make it easier to fix those issues later.

Reply:
I opened https://github.com/wordpress-mobile/WordPress-iOS/issues/17231 in this regard.